packages/playground/src/components/connect_wallet_landing.vue

```diff
@@ -99,6 +97,19 @@ export default {
   setup() {
     const theme = useTheme();
     const baseUrl = import.meta.env.BASE_URL;
+    const network = process.env.NETWORK || (window as any).env.NETWORK;
+
+    const getCapacityHref = () => {
+      let href = `https://dashboard.${network}.grid.tf/explorer/statistics`;
+      if (network === "main") {
+        href = "https://dashboard.grid.tf/explorer/statistics";
+        return href;
+      }
+
+      return href;
+    };
+
+    const capacityURL = getCapacityHref();
```

Comment:
what if we did something like that.

```suggestion

const getCapacityHref = (network: string) => {
  if (network === "main"){
    network = ""
  }
  const = statisticLink = `https://dashboard.${network}.grid.tf/explorer/statistics`;
  return statisticLink;
};

const capacityURL = getCapacityHref();
```

in this example we change the value of the network if the network is equal to "main", then return the URL, also we have an enum called NETWORK maybe we can use it instead of a hard-coded string.

what do you think?

Reply:
URL will be corrupted as in mainnet case will be "https://dashboard..grid.tf/explorer/statistics"

Reply:
It will have two dots given that network is empty

Reply:
indeed, then we can correct it to set the default value of the statisticLink to be the one for the main net, then return it.